simulation/halsim_lowfi/build.gradle

```diff
@@ -0,0 +1,58 @@
+description = "A simulation shared object that publishes basic robot I/O to NetworkTables."
+
+apply plugin: 'edu.wpi.first.NativeUtils'
+apply plugin: 'visual-studio'
+apply plugin: 'cpp'
+
+if (!project.hasProperty('onlyAthena')) {
+    ext.skipAthena = true
+
+    apply from: "../../config.gradle"
+
+
+    model {
+        dependencyConfigs {
+            ntcore(DependencyConfig) {
+                groupId = 'edu.wpi.first.ntcore'
+                artifactId = 'ntcore-cpp'
+                headerClassifier = 'headers'
+                ext = 'zip'
+                version = '4.+'
+                sharedConfigs = [ halsim_lowfi: [] ]
+            }
+			wpiutil(DependencyConfig) {
```

Comment:
Formatting.

Reply:
This was taken from the existing simulation extensions, I'll update all of them since they're formatted the same in master

Reply:
Nevermind, I seem to be mistaken